src/bpftrace.cpp

```diff
@@ -1315,13 +1292,13 @@ int BPFtrace::clear_map(const BpfMap &map)
 
   // snapshot keys, then operate on them
   std::vector<std::vector<uint8_t>> keys;
-  while (bpf_get_next_key(map.fd, old_key.data(), key.data()) == 0) {
+  while (bpf_get_next_key(map.fd(), old_key.data(), key.data()) == 0) {
```

Comment:
Is the call to `bpf_map__fd(bpf_map_);` cheap? If not, maybe save the map fd outside these while loops into a local var.